client/app/components/category_list.vue

```diff
@@ -1,5 +1,94 @@
 <template lang="pug">
-    main(role='contentinfo')
-        h1 {{ 'my_accounts category title' | t }}
-        article {{ 'my_accounts coming_soon' | t }}
+    extends ../templates/layout.pug
+
+    block content
+        cozy-dialog(v-for="item in dialogs", v-bind:item="item", v-on:success="onSuccess", v-on:error="onError", v-on:submit="onSubmit", v-on:close="onCloseDialog")
```

Comment:
IMHO, we have no need for `onError`, `onSubmit`, `onSuccess` props. Only `onClose` can be useful to hook the closing. For the rest, it's all the concerns of the component that will be embed in the dialog.

Also, I still think the dialog component must be placed in the `app.vue`, cause it's the application responsibility to invoke them from the URL, not the repsonsibility of a sub-component.